### PR TITLE
Update autoprofix config to reflect current browser support.

### PR DESCRIPTION
**Pull request by dbatiste**
I _think_ this is what we want.

To give an idea of the raw size change:
```
bsi.css          111KB -> 97KB (-12.6%)
bsi.min.css      89KB -> 79KB (-11.2%)
```

### CODE REVIEW

File: postcss.config.json [hunk 1]
```diff
@@ -3,6 +3,6 @@
 	"input": "./dist/bsi.css",
 	"dir": "./dist/",
 	"autoprefixer": {
-		"browsers": "Last 4 versions, Firefox ESR, IE >= 9, Safari >= 6"
+		"browsers": "Last 1 versions, Firefox ESR, IE >= 11"
```
Comment (dlockhart) [inline postcss.config.json]: The only worry would be Slightly older versions of Safari (9 maybe?). But the savings are significant.

Comment (dbatiste) [inline postcss.config.json]: Yeah.  So, in terms of size, we can get most of the benefit by just going `Last 3 versions, Firefox ESR, IE >= 11, Safari >= 9`...

```
bsi.css          111KB -> 99KB
bsi.min.css      89KB -> 81KB
```

That would be a safer change, and get us most of the benefit.

Comment (dbatiste) [inline postcss.config.json]: Of course, compression makes these differences less significant, but it gives an idea.

Comment (dlockhart) [inline postcss.config.json]: For sure. I'd feel more comfortable with that.

**Pull request merged**